pycbc/inference/sampler.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2016  Christopher M. Biwer
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+"""
+This modules provides classes and functions for using different sampler
+packages for parameter estimation.
+"""
+
+import kombine
```

Comment:
Can we add this as an optional dependency? I"d like us to start constricting the *required* packages for PyCBC and given that people may wan to use different samplers, it seems sane to me to allow people to only have one installed. It seems like the right place to check is either in the KombineSampler initializer, along with a useful error. What do you think?